libraries/tlv-account-resolution/src/state.rs

```diff
@@ -267,6 +228,88 @@ mod tests {
         );
     }
 
+    #[test]
+    fn init_with_metas_and_seeds() {
```

Comment:
All of your tests are really complete and contain non-seeds, so to simplify maintenance you could probably just remove the old tests!

Reply:
Dropped any duplicates but provided separate tests for `init_with_account_metas` and `init_with_account_infos`, which aren't used when seeds are involved

Reply:
Actually, the infos one is used in `init_mixed`, but a simple "does this work" test shouldn't be much clutter